cc_licenses/templates/includes/red_alert.html

```diff
@@ -1,6 +1,6 @@
 <div id="red_alert" class="padding-bigger">
   <p class="has-text-black body-big">
-    <i class="icon">info</i>
+    <i class="icon padding-bottom-smaller padding-right-big">info</i>
```

Comment:
![image](https://user-images.githubusercontent.com/2788679/89789624-d5aa4980-daee-11ea-9ad2-52f70cb5cec0.png)

On a narrower window, the wrapped text doesn't seem to align with either the icon or text above. I'm guessing it'd look better aligned with the text, but if the style guide has another opinion, I guess we should go with that.

Reply:
gave `p` tag a display of `flex`